app/src/components/mode-specific/desktop/MySources/Sources/InstructionsModal/InstructionsHeader.tsx

```diff
@@ -0,0 +1,58 @@
+import { Avatar, Col, Row, Typography } from "antd";
+import { RQButton } from "lib/design-system/components";
+import React from "react";
+
+interface InstructionsHeaderProps {
+  icon: string;
+  heading: string;
+  description: string;
+  setShowInstructions: (value: boolean) => void;
+  RightComponent?: React.ReactNode;
```

Comment:
Please add a doc comment for what this `RightComponent` is for, or we can give some better name.
```ts
/**  ......   */
RightComponent?: React.ReactNode;
```

or we can also design this as
```jsx
<InstructionsHeader {...props}>
  {children}
</InstructionsHeader>
```